src/index.html

```diff
@@ -11,6 +11,39 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="search-form"
+    method="get"
+    action="/"
+    data-qa="big"
+    >
+      <img src="./images/Search.svg"
+      alt="search-icon"
+      class="search-icon"
+      >
+      <input type="search"
+      class="input"
+      name="searchBar"
+      id="searchBar"
+      placeholder="Try &quot;Los Angeles&quot;"
+      data-qa="keypress"
+      >
+    </form>
+
+    <form class="search-form search-form__small"
+    method="get"
```

Comment:
typically we use uppercase when it comes to methods
```suggestion
    method="GET"
```